bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -52,7 +61,7 @@ logging.info("Clustering coinc triggers (exclusive of zerolag)")
 e = e.cluster(args.cluster_window)
 
 logging.info("Dumping foreground triggers")
-f = h5py.File(args.output_file, "w")
+f = h5py.fw(args.output_file)
```

Comment:
Sure but shouldn't the change read `f = fw(args.output_file)` rather than `f = h5py.fw(args.output_file)`? Or are you making fw an attribute of h5py anywhere?